scripts/run_Kraken2-report.R

```diff
@@ -33,13 +33,13 @@ runReport <- function(reportFile,
     output_options = list(self_contained = selfContained),
     params = list(kraken_output = kraken_output,
                   sample_name = sample_name
-                  ),
+    ),
     quiet = quiet
   )
   
-#   if(dir.exists(file.path(sample_dir, sample_name))) {
-#     unlink(file.path(sample_dir, sample_name), recursive = TRUE)
-#   }
+  #   if(dir.exists(file.path(sample_dir, sample_name))) {
+  #     unlink(file.path(sample_dir, sample_name), recursive = TRUE)
+  #   }
   
 }
 
```

Comment:
You could remove all these changes as they are only non-functional whitespace changes.